README.md

```diff
@@ -63,60 +63,77 @@ Optimization simply use Ax.
 - scipy
 - multiple-dispatch
 
+### Prerequisite only for MacOS users:
```

Comment:
There shouldn't be a need to do this for Apple Silicon (unless the user wants to use MPS, but that's a separate story)
```suggestion
### Prerequisite only for MacOS users with Intel processors:
```